qcodes/station.py

```diff
@@ -684,3 +716,45 @@ def update_schema_file(
                 [qcodes.instrument_drivers] + additional_instrument_modules)
         ))
     )
+
+
+def _merge_yamls(*yamls: Union[str, Path]) -> IO[str]:
```

Comment:
Should we consider not making this a private method and add an option to write to disk? I could see it being useful as a tool to consolidate many yaml files.  If we do that maybe pull into it's own file in utils.

Reply:
I also think writing to disk could be potentially useful. Perhaps having a boolean argument like write_to_disk with default of False.

Reply:
@trevormorgan do you mean to making it a public method as it is already a private function? Though for the time being I think it can be private and may be we can think of making this public when the need arises.